extension.json

```diff
@@ -974,8 +1006,16 @@
             "type": "boolean"
           },
           "amount": {
-            "type": "number",
-            "minimum": 0
+            "oneOf": [
+              {
+                "type": "number",
+                "minimum": 0
```

Comment:
I want to discuss if this should remain `0`. It _seems_ like it should given backwards compatibility.

Reply:
Yeah, I think it's fine at 0 and, yeah, we can't increase it without it being a breaking change.